migrations/20240215073437-expense-pipeline-index.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS "expenses__collective_id_status"
+      ON "Expenses"("CollectiveId", "status")
+      WHERE "deletedAt" IS NULL
+    `);
+
+    await queryInterface.sequelize.query(`
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS "collectives__host_collective_id_approved"
+      ON "Collectives"("HostCollectiveId")
+      WHERE "HostCollectiveId" IS NOT NULL AND "deletedAt" IS NULL AND "approvedAt" IS NOT NULL
+    `);
+
+    // Related to security check and findRelatedUsersByConnectedAccounts
+    await queryInterface.sequelize.query(`
+      CREATE INDEX CONCURRENTLY IF NOT EXISTS "connected_accounts__username"
+      ON "ConnectedAccounts"("username")
+      WHERE "deletedAt" IS NULL
```

Comment:
Shouldn't we couple username+service? It seems wrong to fetch an account by username without specifying the service.

Reply:
We can